### PR TITLE
Properly initialize request if URL is null

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
         <groupId>com.google.guava</groupId>
         <artifactId>guava-jdk5</artifactId>
         <version>${project.guava.version}</version>
-      </dependency>            
+      </dependency>
```
Comment (anthmgoogle) [inline pom.xml]: Perhaps do this whitespace cleanup in a separate PR.

**Comment by anthmgoogle** [comment]
Looks good to me (LGTM). Just separate out the whitespace cleanup since it is in an unrelated file.

**Comment by pongad** [comment]
Separated out the whitespace clean up.

**Comment by anthmgoogle** [comment]
LGTM.

**Comment by aozarov** [comment]
Thank you. can you push it to maven?
